### ISSUE TITLE
command 'wallaby.start' not found

### ISSUE THREAD

**Issue by snehalp29** [wallabyjs/public#1779]
### Issue description or question
wallaby won't start error message 
command 'wallaby.start' not found
### Wallaby.js configuration file

```javascript
var wallabyWebpack = require('wallaby-webpack');
var path = require('path');

var compilerOptions = Object.assign(
  require('./tsconfig.json').compilerOptions,
  require('./src/tsconfig.spec.json').compilerOptions);

compilerOptions.module = 'CommonJs';

module.exports = function (wallaby) {

  var webpackPostprocessor = wallabyWebpack({
    entryPatterns: [
      'src/wallabyTest.js',
      'src/**/*spec.js'
    ],

    module: {
      rules: [{
          test: /\.css$/,
          loader: ['raw-loader', 'css-loader']
        },
        {
          test: /\.html$/,
          loader: 'raw-loader'
        },
        {
          test: /\.ts$/,
          loader: '@ngtools/webpack',
          include: /node_modules/,
          query: {
            tsConfigPath: 'tsconfig.json'
          }
        },
        {
          test: /\.js$/,
          loader: 'angular2-template-loader',
          exclude: /node_modules/
        },
        {
          test: /\.json$/,
          loader: 'json-loader'
        },

        {
          test: /\.styl$/,
          loaders: ['raw-loader', 'stylus-loader']
        }, {
          test: /\.less$/,
          loaders: ['raw-loader', 'less-loader']
        }, {
          exclude: [path.join(process.cwd(), 'src/styles/styles.scss'), path.join(process.cwd(), 'src/assets/**/*.*')],
          test: /\.scss$|\.sass$/,
          use: [
            'exports-loader?module.exports.toString()',
            {
              loader: 'css-loader',
              options: {
                sourceMap: false,
                importLoaders: 1
              }
            },
            {
              loader: 'sass-loader',
              options: {
                sourceMap: false,
                precision: 8,
                includePaths: [
                  'node_modules/pru-strap/dist/sass',
                  'node_modules/bootstrap-sass/assets/stylesheets/bootstrap',
                  'node_modules/font-awesome/scss',
                  'node_modules/bootstrap-datepicker',
                  'node_modules/global-navigation/src/styles/styles.scss'
                ]
              }
            }
          ]
        }, {
          include: [path.join(process.cwd(), 'src/styles/styles.scss'), path.join(process.cwd(), 'src/assets/**/*.*')],
          test: /\.scss$|\.sass$/,
          use: [
            'exports-loader?module.exports.toString()',
            {
              loader: 'css-loader',
              options: {
                sourceMap: false,
                importLoaders: 1
              }
            },
            {
              loader: 'sass-loader',
              options: {
                sourceMap: false,
                precision: 8,
                includePaths: [
                  'node_modules/pru-strap/dist/sass',
                  'node_modules/bootstrap-sass/assets/stylesheets/bootstrap',
                  'node_modules/font-awesome/scss',
                  'node_modules/bootstrap-datepicker',
                  'node_modules/global-navigation/src/styles/styles.scss'
                ]
              }
            }
          ]
        },
        {
          test: /\.less$/,
          loaders: ['raw-loader', 'less-loader']
        },
        {
          test: /\.(jpg|png)$/,
          loader: 'url-loader?limit=128000'
        }
      ]
    },

    resolve: {
      extensions: ['.js', '.ts'],
      modules: [
        path.join(wallaby.projectCacheDir, 'src/app'),
        path.join(wallaby.projectCacheDir, 'src'),
        'node_modules'
      ],
      alias: {
        mocks: path.join(wallaby.projectCacheDir, 'src/mocks/')
      }

    },
    node: {
      fs: 'empty',
      net: 'empty',
      tls: 'empty',
      dns: 'empty'
    }
  });

  return {
    files: [{
        pattern: 'src/**/*.+(ts|js|css|less|scss|sass|styl|html|json|svg|jpg)',
        load: false
      },
      {
        pattern: 'node_modules/​​global-navigation/**/*.+(ts|js)',
      },
      {
        pattern: 'node_modules/pru-strap/dist/assets/pru-strap.min.js',
        instrument: false
      },
      {
        pattern: 'node_modules/bootstrap-datepicker/dist/js/bootstrap-datepicker.js',
        instrument: false
      },

      {
        pattern: 'src/**/*.d.ts',
        ignore: true
      },
      {
        pattern: 'src/**/*spec.ts',
        ignore: true
      }
    ],

    tests: [{
        pattern: 'src/**/*spec.ts',
        load: false
      },
      {
        pattern: 'src/**/*e2e-spec.ts',
        ignore: true
      }
    ],

    testFramework: 'jasmine',

    compilers: {
      '**/*.ts': wallaby.compilers.typeScript(compilerOptions)
    },

    middleware: function (app, express) {
      var path = require('path');
      app.use('/favicon.ico', express.static(path.join(__dirname, 'src/favicon.ico')));
      app.use('/assets', express.static(path.join(__dirname, 'src/assets')));
    },

    env: {
      kind: 'electron'
    },

    postprocessor: webpackPostprocessor,

    setup: function () {
      window.__moduleBundler.loadTests();
    },

    debug: true
  };
};

```

### Code editor or IDE name and version

Visual Studio Code v1.25.1

### OS name and version

Windows 7 Enterprise

**Comment by snehalp29** [comment]
I have tried uninstall and reinstall VScode plugin and uninstall and  reinstall VSCode

**Comment by snehalp29** [comment]
![image](https://user-images.githubusercontent.com/11776039/43636519-4deea878-96e0-11e8-9ff4-787a8bb94754.png)

**Comment by ArtemGovorov** [comment]
Please share the wallaby.json file content that the last error on your screenshot points to. 
Are you behind some corporate proxy?

**Comment by snehalp29** [comment]
<HTML><HEAD>
<TITLE>Access Denied</TITLE>
</HEAD>
<BODY>
<FONT face="Helvetica">
<big><strong></strong></big><BR>
</FONT>
<blockquote>
<TABLE border=0 cellPadding=1 width="80%">
<TR><TD>
<FONT face="Helvetica">
<big>Access Denied (authentication_failed)</big>
<BR>
<BR>
</FONT>
</TD></TR>
<TR><TD>
<FONT face="Helvetica">
Your credentials could not be authenticated: "Credentials are missing.". You will not be permitted access until your credentials can be verified.
</FONT>
</TD></TR>
<TR><TD>
<FONT face="Helvetica">
This is typically caused by an incorrect username and/or password, but could also be caused by network problems. Pru401
</FONT>
</TD></TR>
<TR><TD>
<FONT face="Helvetica" SIZE=2>
<BR>
For assistance, contact your local help desk.
</FONT>
</TD></TR>
</TABLE>
</blockquote>
</FONT>
</BODY></HTML>

**Comment by snehalp29** [comment]
this was working for me couple of days back

**Comment by ArtemGovorov** [comment]
Looks like you are now behind some corporate proxy that stops Wallaby from installing its core.

**Comment by snehalp29** [comment]
whats should i try now ?

**Comment by ArtemGovorov** [comment]
Please try the solution described at https://github.com/wallabyjs/public/issues/1123.

**Comment by snehalp29** [comment]
hey thanks this is working now

**Comment by sgariepy** [comment]
I have the same problem, except I don't have any config file and I'm not behind a proxy.  I have to reinstall WallabyJS Extension in VS Code every 2-3 times I use it.  Other commands like `wallaby.showCommandPalette` or `wallaby.forceCoreUpdate` are also unavailable.

VSCode version 1.44.2 on macOS 10.14.6.

**Comment by smcenlly** [comment]
@sgariepy - thanks for letting us know. We've made some changes to our updater logic (available in the Wallaby for VS Code extension v1.0.197) to attempt to address this issue for you. It was previously possible to stop VS Code while Wallaby was extracting its updated files which could lead to a corruption in your install. Could you please update to the latest version and let us know if you're still having the problem. 

If you still have problems with the latest version, please let us know and also provide us ZIP and upload a copy of your wallaby extension to analyze (`~/.vscode/extensions/wallabyjs.wallaby-vscode-${VERSION}`).
